Fake.sln

```diff
@@ -170,6 +170,8 @@ Project("{6EC3EE1D-3C4E-46DD-8F32-0CC8E7565705}") = "Fake.Tools.Octo", "src\app\
 EndProject
 Project("{6EC3EE1D-3C4E-46DD-8F32-0CC8E7565705}") = "Fake.Core.Vault", "src\app\Fake.Core.Vault\Fake.Core.Vault.fsproj", "{AAAF92C5-C40D-40B8-84BA-137DF0E98B56}"
 EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Fake.Tools.Squirrel", "src\app\Fake.Tools.Squirrel\Fake.Tools.Squirrel.fsproj", "{3DEF2E95-4BF8-413F-930E-32103A39364A}"
```

Comment:
Can we change the guid here to `{6EC3EE1D-3C4E-46DD-8F32-0CC8E7565705}` instead of `{F2A71F9B-5D33-465A-A702-920D77279786}`?

Reply:
Yes, no problem I just followed the PR guidelines of running `dotnet sln Fake.sln add src/app/Fake.*/Fake.*.fsproj`

Reply:
Yes no idea about those guids but as soon as you open the sln in VS it will update the guid, don't ask me I have no idea what they are doing.

Reply:
Visual Studio dark magic 🧙🏻‍

Reply:
probably this: https://github.com/dotnet/project-system/issues/1821